lib/text/text_engine.js

```diff
@@ -196,9 +197,14 @@ shaka.text.TextEngine = class {
       vttOffset: vttOffset,
     };
 
+    let finalUri = null;
```

Comment:
What's the purpose of this variable? As far as I can tell, all it does it typecast an uri of `undefined` to `null`.
If that's the case, couldn't you just put `uri = null` in the function declaration above, or `uri || null` inside the call to `parseMedia` below?

Reply:
Changed! Thanks!